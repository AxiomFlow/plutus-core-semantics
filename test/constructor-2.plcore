(module Foo (imported) (exported () ()) (local () ())
  (define result
    (case (con Nat.S (con Nat.Z))
          (cl Nat.S (x) (con Nat.Z))
          (cl Nat.Z ()  -1))))
