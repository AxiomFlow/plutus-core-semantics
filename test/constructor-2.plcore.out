<T> <k> . </k> <env> result |-> ( con Nat . Z .TmList ) </env> <subst> .Map </subst> </T>
